 btrfs-tools (3.17-netgear1) unstable; urgency=low
 .
   * New upstream release.
   * Move scrub status files to tmpfs.
Author: Justin Maggard <justin.maggard@netgear.com>

---

--- btrfs-tools-3.17.orig/cmds-scrub.c
+++ btrfs-tools-3.17/cmds-scrub.c
@@ -48,8 +48,8 @@ static const char * const scrub_cmd_grou
 	NULL
 };
 
-#define SCRUB_DATA_FILE "/var/lib/btrfs/scrub.status"
-#define SCRUB_PROGRESS_SOCKET_PATH "/var/lib/btrfs/scrub.progress"
+#define SCRUB_DATA_FILE "/run/btrfs/scrub.status"
+#define SCRUB_PROGRESS_SOCKET_PATH "/run/btrfs/scrub.progress"
 #define SCRUB_FILE_VERSION_PREFIX "scrub status"
 #define SCRUB_FILE_VERSION "1"
 
